crates/driver/openapi.yml

```diff
@@ -334,41 +334,48 @@ components:
     SolveResponse:
       description: |
         Response of the solve endpoint.
-      type: array
-      items:
-        type: object
-        properties:
-          solutionId:
-            description: |
-              The unique identifier of the solution.
-
-              This id is used to identify the solution when executing it.
-            type: integer
-          score:
-            description: |
-              The objective value of the solution.
-            type: number
-          submissionAddress:
-            description: |
-              The address that will be used to submit the solution.
-            $ref: "#/components/schemas/Address"
-          orders:
-            description: executed orders
-            type: array
-            items:
-              type: object
-              description: |
-                Mapping of order uid to net executed amount (including all fees).
-              additionalProperties:
-                $ref: "#/components/schemas/TokenAmount"
-          clearingPrices:
-            description: |
-              Mapping of hex token address to price.
-
-              The prices of tokens for settled user orders as passed to the settlement contract.
+      type: object
+      properties:
+        solutions:
+          type: array
+          items:
             type: object
-            additionalProperties:
-              $ref: "#/components/schemas/BigUint"
+            properties:
+              solutionId:
+                description: |
+                  The unique identifier of the solution.
+    
+                  This id is used to identify the solution when executing it.
+                type: string
+                example: "1"
+              score:
+                description: |
+                  The objective value of the solution.
+                type: string
+                example: "100"
+              submissionAddress:
+                description: |
+                  The address that will be used to submit the solution.
+                $ref: "#/components/schemas/Address"
+              orders:
+                type: object
```

Comment:
https://github.com/cowprotocol/services/blob/main/crates/autopilot/src/infra/solvers/dto/solve.rs#L146

Order uid is missing